cmd/preflight/cli/stdout_results.go

```diff
@@ -90,6 +92,7 @@ func outputResult(analyzeResult *analyzerunner.AnalyzeResult) bool {
 		fmt.Printf("      --- %s\n", analyzeResult.Message)
 	} else if analyzeResult.IsFail {
 		fmt.Printf("   --- FAIL: %s\n", analyzeResult.Title)
+		fmt.Printf("      --- Strict: %t\n", analyzeResult.Strict)
```

Comment:
maybe only output this if strict is true?
something like
` --- THIS FAILURE BLOCKS INSTALLATION` instead of ` --- Strict: true`

That also means we wouldn't change the current output without people having updated

Reply:
Since `strict` can be consumed in any manner,
We (kots) are blocking the deployments when preflight's result `strict` is true
but for any other clients consuming support-bundle, `strict` can mean it different.

We would need a better message for the flag

Reply:
I still think "don't change the current output" is a better option

Reply:
Changed it. strict flags are displayed when strict is only `true`